config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -119,6 +119,10 @@ def build_test(cloud='aws',
     if instance_groups_overrides is None:
         instance_groups_overrides = []
 
+    kops_image = None
+    kops_ssh_user = None
+    kops_ssh_key_path = None
```

Comment:
I'll probably do this in a separate PR. AFAIK this script don't rely on the repo dependencies. cc @hakman

Reply:
It might, do you see any issue with the changes, they seem minor?

Reply:
It got flagged by the linter after the upgrade